config/eventhub/config.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eventhub
+
+import (
+	"github.com/crossplane/terrajet/pkg/config"
+
+	"github.com/crossplane-contrib/provider-jet-azure/apis/rconfig"
+	"github.com/crossplane-contrib/provider-jet-azure/config/common"
+)
+
+// Configure configures resource group
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("azurerm_eventhub_namespace", func(r *config.Resource) {
+		r.Version = common.VersionV1Alpha2
+		r.Kind = "EventHubNamespace"
+		r.ShortGroup = "eventhub"
+		r.References = config.References{
+			"resource_group_name": config.Reference{
```

Comment:
I think we are configuring `resource_group` references here:
https://github.com/crossplane-contrib/provider-jet-azure/blob/dba5579884acf113e314e25aa6c4d64b6c15415c/config/common/common.go#L158
Do we need this?